### PR TITLE
GET/api/articles/article-id/comments

### PR DESCRIPTION
**Pull request by PJ31SPR**

### CODE REVIEW

File: __tests__/integration.test.js [hunk 1]
```diff
@@ -120,15 +120,42 @@ describe("/api/articles", () => {
...
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad Request");
+      });
   });
-});
\ No newline at end of file
+  test('GET 404: Not Found when resource is not found', () => {
+    return request(app)
+      .get('/api/articles/99999999/comments')
```
Comment (nc-verity-gregory) [inline __tests__/integration.test.js:153]: Be careful with ids like this, you can run into errors for such large numbers. As we're on a test db a solid 99 would work fine.

File: __tests__/integration.test.js [hunk 2]
```diff
@@ -120,15 +120,42 @@ describe("/api/articles", () => {
...
-});
\ No newline at end of file
+  test('GET 404: Not Found when resource is not found', () => {
+    return request(app)
+      .get('/api/articles/99999999/comments') 
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Not Found")
+        expect(response.status).toBe(404)
+      })
```
Comment (nc-verity-gregory) [inline __tests__/integration.test.js:158]: What about a brand new article, would that have comments? You need a 200 test for an empty response.

File: models/comments.models.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+exports.selectComments = (id) => {
+// console.log('hello from comments model')
+return db.query(`SELECT comment_id, votes, created_at, author, body, article_id
+FROM comments
+WHERE article_id = $1
+ORDER BY created_at DESC;
+`, [id]).then((response) => {
+// console.log(response.rows, '<-- resp.rows model')
+if(response.rows.length === 0){
```
Comment (nc-verity-gregory) [inline models/comments.models.js:11]: Is it an error if there are no comments on an article?

**Review by nc-verity-gregory — COMMENTED** [review]
Good start on this but you are missing a test for no comments. Its not a problem that an article doesn't have any.
